site/app/views/grading/ElectronicGraderView.php

```diff
@@ -493,7 +493,7 @@ public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_
                 $columns[]     = ["width" => "10%", "title" => "Grading",          "function" => "grading_blind"];
             }
             if ($peer === false) {
-                $columns[]     = ["width" => "15%", "title" => "Total",            "function" => "total"];
+                $columns[]     = ["width" => "15%", "title" => "Total",            "function" => "total",  "sort_type" => "first"];
```

Comment:
Adding sort type of first will only sort by the first name column. This will not sort by auto grading scores.